Resources/doc/cookbook/recipe_custom_action.rst

```diff
@@ -92,7 +92,10 @@ to implement a ``clone`` action.
 
     class CRUDController extends Controller
     {
-        public function cloneAction()
+        /**
+         * @param $id
+         */
+        public function cloneAction($id)
```

Comment:
Yes, just for this.
Here I hesitated between adding its declaration back or removing its use in the error message (which is actually what I did in my own project).